cmake/config/linux-gcc-clang.txt

```diff
@@ -167,6 +167,11 @@ endif ()
 # Release configuration.
 set (c_compiler_flags_release
     -O3                                                 # optimization level
+    -funroll-loops                                      # Unrolls loops
+    -fgcse-las                                          # Eliminates stores followed by a load to the same positions.
+    -fgcse-sm                                           # Moves load/save instructions out of loops, if possible.
+    -fgcse-after-reload                                 # Cleans different fgcse passes up.
```

Comment:
Already enabled by `-O3`?

Reply:
Oh, that one slipped through. In the description of -fgcse-after-reload it doesn't say anything about being enabled automatically, but on the top at O3. Gonna remove that one.